### PR TITLE
Add note about --fs-group option

### PR DESCRIPTION
**Pull request by dmage**
https://bugzilla.redhat.com/show_bug.cgi?id=1447249

/cc @bparees @miminar @legionus

### CODE REVIEW

File: install_config/registry/deploy_registry_existing_clusters.adoc [hunk 1]
```diff
@@ -79,6 +79,9 @@ To see a full list of options that you can specify when creating the registry:
 ----
 $ oc adm registry --help
 ----
+
+Please note that the value for the `--fs-group` option should be agreed with
```
Comment (bparees) [inline install_config/registry/deploy_registry_existing_clusters.adoc]: /should be agreed with/must be permitted by the fsgroup setting of the SCC/ 

(which SCC is used?  restricted?  they need to know which one to look at)

**Review by bparees — CHANGES_REQUESTED** [review]

File: install_config/registry/deploy_registry_existing_clusters.adoc (hunk 1 above)
Comment (dmage) [reply install_config/registry/deploy_registry_existing_clusters.adoc]: In general you don't know which one will be used.

If you use `oc cluster up`, the restricted SCC is applied and in this case you should choose a value from the range:
```
$ oc -n default get ns default -o template --template='{{index .metadata.annotations "openshift.io/sa.scc.supplemental-groups"}}{{"\n"}}'
1000000000/10000
```

Or you can create an SCC with the specific supplemental group and add the service account to it. And this SCC will be applied to the registry automagically if you create the deployment config with this group.

Comment (bparees) [reply install_config/registry/deploy_registry_existing_clusters.adoc]: ok, i'd just hand wave something like `the value for --fs-group must be permitted by the SCC used by the registry (typically, the restricted SCC)`

**Comment by bparees** [comment]
lgtm, @openshift/team-documentation ptal

**Comment by bmcelvee** [comment]
Thanks, @bparees! I will take a look at this in a bit.

**Comment by bmcelvee** [comment]
This applies to OCP 3.6 forward, correct?

**Comment by bparees** [comment]
yes

**Pull request merged**